### PR TITLE
feat: WEB-253 Update pulse index page

### PR DESCRIPTION
**Pull request by carlagn**
Fixes [#WEB-253](https://linear.app/prisma-company/issue/WEB-253/update-pulse-landing-page-in-docs)

### CODE REVIEW

**Review by mhessdev — APPROVED** [review]

File: content/400-pulse/index.mdx [hunk 1]
```diff
@@ -1,19 +1,100 @@
...
+  />
+  <LinkCard
+    icon="fa-brands fa-github"
+    title="Examples"
+    desc="Explore our ready-to-run examples using Accelerate."
+    link="https://github.com/prisma/prisma-examples/"
+  />
+</Grid>
+<br/>
+<LinkCard
```
Comment (meletj) [inline content/400-pulse/index.mdx]: remove speed test section - not applicable for pulse page

**Comment by Techboidesign** [comment]
@carlagn maybe we can have it also be **justify: center** like in accelerate, and make the box go all the width, also like in accelerate: 

image
image

File: content/400-pulse/index.mdx [hunk 2]
```diff
@@ -1,19 +1,100 @@
...
+
+<IconTitle heading="h1" icon="fa-solid fa-signal-stream" iconColor="#16A394">Prisma Pulse</IconTitle>
+
+[Prisma Pulse](https://www.prisma.io/data-platform/pulse) enables real-time database events with type-safe Prisma Client subscriptions.
+
+<Grid>
+  <LinkCard
+    icon="fa-solid fa-arrow-right"
+    title="Get started"
+    desc="Add Accelerate to your app to use global database caching & connection pooling."
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
    desc="Add Pulse to your app in minutes to get real-time updates from your database."
```

File: content/400-pulse/index.mdx [hunk 3]
```diff
@@ -1,19 +1,100 @@
...
+    link="/accelerate/getting-started"
+  />
+  <LinkCard
+    icon="fa-brands fa-github"
+    title="Examples"
+    desc="Explore our ready-to-run examples using Accelerate."
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
    desc="Explore our ready-to-run examples using Pulse."
```

File: content/400-pulse/index.mdx [hunk 4]
```diff
@@ -1,19 +1,100 @@
...
+    link="https://github.com/prisma/prisma-examples/"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
    link="https://github.com/prisma/prisma-examples?tab=readme-ov-file#prisma-pulse"
```

File: content/400-pulse/index.mdx [hunk 5]
```diff
@@ -1,19 +1,100 @@
...
+<IconTitle heading="h1" icon="fa-solid fa-signal-stream" iconColor="#16A394">Prisma Pulse</IconTitle>
+
+[Prisma Pulse](https://www.prisma.io/data-platform/pulse) enables real-time database events with type-safe Prisma Client subscriptions.
+
+<Grid>
+  <LinkCard
+    icon="fa-solid fa-arrow-right"
+    title="Get started"
+    desc="Add Accelerate to your app to use global database caching & connection pooling."
+    link="/accelerate/getting-started"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
    link="/pulse/getting-started"
```

File: content/400-pulse/index.mdx [hunk 6]
```diff
@@ -1,19 +1,100 @@
...
+    highlight
+    icon="fa-solid fa-bolt"
+    title="Speed Test"
+    desc="Run the speed test to see how Prisma Accelerate can make your app faster."
+    link="https://accelerate-speed-test.prisma.io/"
+/>
+
+## Supported databases and providers
+
+Short description text for this section.
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
Pulse works with your existing database and providers.
```

File: content/400-pulse/index.mdx [hunk 7]
```diff
@@ -1,19 +1,100 @@
...
+
+<List framed style={{ alignItems: `center`, width: `fit-content`}}>
+    <SquareLogo
+        url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgresql"
+        image={'/img/technologies/postgresqlsimple.svg'}
+        tech="Postgres SQL"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
        tech="PostgreSQL"
```

File: content/400-pulse/index.mdx [hunk 8]
```diff
@@ -1,19 +1,100 @@
...
+    link="https://accelerate-speed-test.prisma.io/"
+/>
+
+## Supported databases and providers
+
+Short description text for this section.
+
+<List framed style={{ alignItems: `center`, width: `fit-content`}}>
+    <SquareLogo
+        url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgresql"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
        url="/pulse/database-setup/general-database-instructions"
```

File: content/400-pulse/index.mdx [hunk 9]
```diff
@@ -1,19 +1,100 @@
...
+        image={'/img/technologies/postgresqlsimple.svg'}
+        tech="Postgres SQL"
+    />
+    <Separator />
+    <List style={{marginTop: 0}} split>
+        <SquareLogo
+            url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-mysql"
+            image={'/img/technologies/mysqlsimple.svg'}
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            image={'/img/technologies/gcp.svg'}
```

File: content/400-pulse/index.mdx [hunk 10]
```diff
@@ -1,19 +1,100 @@
...
+            tech="MySQL"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            tech="GCP"
```

File: content/400-pulse/index.mdx [hunk 11]
```diff
@@ -1,19 +1,100 @@
...
+<List framed style={{ alignItems: `center`, width: `fit-content`}}>
+    <SquareLogo
+        url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgresql"
+        image={'/img/technologies/postgresqlsimple.svg'}
+        tech="Postgres SQL"
+    />
+    <Separator />
+    <List style={{marginTop: 0}} split>
+        <SquareLogo
+            url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-mysql"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            url="pulse/database-setup"
```

File: content/400-pulse/index.mdx [hunk 12]
```diff
@@ -1,19 +1,100 @@
...
+            image={'/img/technologies/mysqlsimple.svg'}
+            tech="MySQL"
+        />
+        <SquareLogo
+            url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-sqlserver"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            url="/pulse/database-setup/aws-rds"
```

File: content/400-pulse/index.mdx [hunk 13]
```diff
@@ -1,19 +1,100 @@
...
+            image={'/img/technologies/sqlserver.svg'}
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            image={'/img/technologies/aws.svg'}
```

File: content/400-pulse/index.mdx [hunk 14]
```diff
@@ -1,19 +1,100 @@
...
+            tech="SQLServer"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            tech="AWS"
```

File: content/400-pulse/index.mdx [hunk 15]
```diff
@@ -1,19 +1,100 @@
...
+        />
+        <SquareLogo
+            url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-planetscale"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            url="/pulse/database-setup/supabase"
```

File: content/400-pulse/index.mdx [hunk 16]
```diff
@@ -1,19 +1,100 @@
...
+            image={'/img/technologies/planetscale.svg'}
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            image={'/img/technologies/supabase.svg'}
```

File: content/400-pulse/index.mdx [hunk 17]
```diff
@@ -1,19 +1,100 @@
...
+            tech="PlanetScale"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            tech="Supabase"
```

File: content/400-pulse/index.mdx [hunk 18]
```diff
@@ -1,19 +1,100 @@
...
+        />
+        <SquareLogo
+            url="/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-cockroachdb"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            url="/pulse/database-setup/neon"
```

File: content/400-pulse/index.mdx [hunk 19]
```diff
@@ -1,19 +1,100 @@
...
+            image={'/img/technologies/cockroachdbgradient.svg'}
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            image={'/img/technologies/neon.svg'}
```

File: content/400-pulse/index.mdx [hunk 20]
```diff
@@ -1,19 +1,100 @@
...
+            tech="Cockroach DB"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            tech="Neon"
```

File: content/400-pulse/index.mdx [hunk 21]
```diff
@@ -1,19 +1,100 @@
...
+        />
+        <SquareLogo
+            url="/getting-started/setup-prisma/start-from-scratch/mongodb-typescript-mongodb"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            url="/pulse/database-setup/railway"
```

File: content/400-pulse/index.mdx [hunk 22]
```diff
@@ -1,19 +1,100 @@
...
+            image={'/img/technologies/mongodbsimple.svg'}
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            image={'/img/technologies/railway.svg'}
```

File: content/400-pulse/index.mdx [hunk 23]
```diff
@@ -1,19 +1,100 @@
...
+            tech="Mongo DB"
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
            tech="Railway"
```

File: content/400-pulse/index.mdx [hunk 24]
```diff
@@ -1,19 +1,100 @@
...
-[Prisma Pulse](https://www.prisma.io/data-platform/pulse) enables real-time database events with type-safe Prisma Client subscriptions.
+import {
+    IconTitle
+} from '@site/src/components/shortcodes/index';
 
-</TopBlock>
+
+<IconTitle heading="h1" icon="fa-solid fa-signal-stream" iconColor="#16A394">Prisma Pulse</IconTitle>
+
+[Prisma Pulse](https://www.prisma.io/data-platform/pulse) enables real-time database events with type-safe Prisma Client subscriptions.
```
Comment (meletj) [inline content/400-pulse/index.mdx]:
```suggestion
[Prisma Pulse](https://www.prisma.io/data-platform/pulse) lets you subscribe to database changes easily and in a type-safe way using Prisma Client.
```

**Comment by meletj** [comment]
Update src/css/gettingStarted.module.scss from https://github.com/prisma/docs/pull/6026

**Pull request merged**